source/js/buyers-guide/index.js

```diff
@@ -1 +1,2 @@
 export { injectReactComponents } from "./inject-react";
+export { bindEventHandlers } from "./template-js-handler";
```

Comment:
this can probably be `export { mobileNavStickinessHandler  as bindEventHandlers } from "./mobile-nav-stickiness-handler"` and then we don't need the separate template-js-handler file

Reply:
I'm trying to follow our existing dir structures with the [main foundation site](https://github.com/mozilla/foundation.mozilla.org/tree/master/source/js/foundation). Let me know your thoughts. I can update my PR accordingly.

Reply:
we can keep this for now, I'm fine with cleaning that up (maybe =D) later